stable/wordpress/templates/ingress.yaml

```diff
@@ -1,28 +1,37 @@
-{{- if .Values.ingress.enabled -}}
+{{- if .Values.ingress.enabled }}
+{{- $rootContext := . }}
```

Comment:
I think you can always get the root context in `$` (e.g. `$.Release.Name`)

Reply:
I never knew that, fixed